README.rst

```diff
@@ -109,13 +109,26 @@ Global options
     `--source-pipeline=videotestsrc`. A script like `press("18")` will change
     videotestsrc's pattern property (see `gst-inspect videotestsrc`).
 
-  x11:<display>
+  x11:<display>,<key_mapping_file>
```

Comment:
Is this a comma because `display` can contain colons? Does display have to have the colon like `:0` or does it have to be without the colon like `0` or can it be either?

If `key_mapping_file` is optional it should read `[,<key_mapping_file>]`. Same goes for `display` I suppose.